File: Java/Program/Gpu.java
public class Gpu extends Komponen {
    private int vramGB;
    private String chipset;

    public Gpu(String merk, String nama, int vramGB, String chipset) {
        super(merk, nama);
        this.vramGB = vramGB;
        this.chipset = chipset;
    }

    public void setVramGB(int vramGB) {
        this.vramGB = vramGB;
    }

    public int getVramGB() {
        return vramGB;
    }

    public void setChipset(String chipset) {
        this.chipset = chipset;
    }

    public String getChipset() {
        return chipset;
    }
}

File: Python/Program/Gpu.py
from Komponen import Komponen

class Gpu(Komponen):
    def __init__(self, merk, nama, vramGB, chipset):
        super().__init__(merk, nama)
        self.vramGB = vramGB
        self.chipset = chipset
        
    def setVramGB(self, vramGB):
        self.vramGB = vramGB
        
    def getVramGB(self):
        return self.vramGB
    
    def setChipset(self, chipset):
        self.chipset = chipset
        
    def getChipset(self):
        return self.chipset
    
